res/js/tx.caretaker.NodeTree.js

```diff
@@ -354,7 +355,7 @@ tx.caretaker.NodeTree = Ext.extend(Ext.tree.TreePanel, {
 				"tx_caretaker[isInInstance]": (node.attributes.type == 'instance' ? 1 : 0),
 				"tx_caretaker[type]": 'test',
 				"tx_caretaker[storagePid]": this.storagePid,
-				"tx_caretaker[returnUrl]": encodeURIComponent(parent.list_frame.document.location)
```

Comment:
Hi,

because the url was encoded two times in backend an than it was not possible to go back or something else.

Issue describtion
* Go to caretaker view in BE
* Edit a test case
* click "go back" (not save or something else) 
* This is not possible at the moment